framework/CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Yii Framework 2 Change Log
 - Enh #11139: `yii\validators\EachValidator` injects specific attribute value in error message parameters (silverfire)
 - Enh #11187: migrate command now generates phpdoc for table migrations (Faryshta)
 - Enh: Added `StringHelper::countWords()` that given a string returns number of words in it (samdark)
+- Enh: migrate command can create foreign keys.
```

Comment:
I mean it could be same tables in both cases:

```
user
====
id
name

product
======
id
name
created_by --> FK to user
updated_by --> FK to user
```